cpp/harvest/nacjd.cc

```diff
@@ -64,13 +64,29 @@ namespace {
         "\t- marc_path: The output MARC file.\n"
         "\n"
         "get_statistics json_path\n"
+        "\t- json_path: The input JSON file.\n"
+        "\n"
+        "get_categories_statistics json_path\n"
         "\t- json_path: The input JSON file.\n");
 }
 
 
 std::vector<std::string> ids_website;
 const unsigned int TIMEOUT_IN_SECONDS(15);
 
+const std::string statistics_categories_array[] = { "National Crime Victimization Survey",
+                                                    "National Incident-Based Reporting System",
+                                                    "FBI Uniform Crime Reporting",
+                                                    "Annual Parole Survey",
+                                                    "Annual Probation Survey",
+                                                    "Annual Survey of Jails",
+                                                    "Mortality in Correctional Institutions",
+                                                    "Federal Court Cases Integrated Database",
+                                                    "Federal Justice Statistics Program",
+                                                    "National Corrections Reporting Program" };
+
+int statistics_count[] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
1) This should not be defined as a global variable, it should be declared locally intead
2) Instead of using a int[] with a hardcoded length, you should use a more dynamic datatype instead, e.g. a std::map<std::string, unsigned> (where the std::string is the name of the category).